lib/config-generator.js

```diff
@@ -331,11 +331,31 @@ ConfigGeneraror.prototype = {
 
         return new Promise(function(resolve, reject) {
             var result = [];
-
+            var isRequire = false;
             jsstana.traverse(ast, function(node) {
-                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?))', node);
+                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?) (call ?.define ?) (call ?.define ? ?) (call ?.define ? ? ?) (call require ?) (call require ? ?) (call require ? ? ?) (call ?.require ?) (call ?.require ? ?) (call ?.require ? ? ?))', node);
```

Comment:
Hey Chema,

Yeah, this may happen. To solve it, one solution is to pass two variables - `src` namespace and `dest` namespace. Then, if one wants to rename `Liferay.define` to `MyCompany.define`, he will have to specify both variables.

We can do it, but if we do it, we will solve a problem we don't have and it doesn't look we will ever have. Of course, the statement above assumes files in Liferay Portal are transpiled only once and then overwritten. This is still like this, isn't it?
If so, then I would say let's cover what is needed and not catch Liferay.define or Liferay.require.

Thanks,